### PR TITLE
Block Sync: Syncing historical blocks using IPLD or Shrex

### PR DESCRIPTION
**Pull request by derrandz**
## Overview

This PR adds testing/benchmarking scenarios for syncing historical blocks for IPLD and Shrex Getters

## Checklist

- [ ] New and updated code has appropriate documentation
- [ ] New and updated code has new and/or updated testing
- [ ] Required CI checks are passing
- [ ] Visual proof for any user facing features like CLI or documentation updates
- [ ] Linked issues closed with keywords

### CODE REVIEW

File: tests/helpers/block-sync/historical/run_bridge.go [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+)
+
+func RunBridgeNode(runenv *runtime.RunEnv, initCtx *run.InitContext) error {
+	ctx, cancel := context.WithTimeout(
+		context.Background(),
+		time.Minute*time.Duration(runenv.IntParam("execution-time")),
+	)
+	defer cancel()
+
+	// runenv.D().Counter("bridge_nodes_reached_target_height").Clear()
```
Comment (Bidon15) [inline tests/helpers/block-sync/historical/run_bridge.go:24]: Oops :D

File: compositions/cluster-k8s/block-sync/historical/64-square-size/1-3-32-ipld.toml [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+  execution-time = "20"
+  persistent-peers = "1"
+  submit-times = "10"
+  msg-size = "500000"
+  validator = "1"
+  bridge = "3"
+  full = "16"
+  historical = "16"
+  block-height = "30"
+  otel-collector-address = "178.128.163.171:4318"
```
Comment (Bidon15) [inline compositions/cluster-k8s/block-sync/historical/64-square-size/1-3-32-ipld.toml:25]: Can you please give a comment what is this IP? 
I suppose it's a DO one?

File: tests/helpers/block-sync/historical/run_historical_full.go [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+		eh.Commit.BlockID.Hash.String())
+
+	state, err := nd.HeaderServ.SyncState(ctx)
+	if err != nil {
+		return err
+	}
+	if !state.Finished() {
+		return fmt.Errorf("full node is still syncing the past")
+	}
+	<-time.After(time.Minute * 2)
```
Comment (Bidon15) [inline tests/helpers/block-sync/historical/run_historical_full.go:158]: Is it worth another variable in the manifest?

File: tests/helpers/block-sync/historical/run_historical_full.go [hunk 4]
```diff
@@ -0,0 +1,175 @@
...
+	_, err = syncclient.SignalEntry(ctx, testkit.FinishState)
+	err = nd.Stop(ctx)
+	if err != nil {
+		return err
+	}
+
+	_, err = syncclient.SignalEntry(ctx, testkit.FinishState)
+	if err != nil {
+		return err
+	}
```
Comment (Bidon15) [inline tests/helpers/block-sync/historical/run_historical_full.go:172]: Why do we signal twice here? 🧐

**Review by Bidon15 — CHANGES_REQUESTED** [review]
Awesome job! Some comments and it's ready to be merged!

File: tests/helpers/block-sync/historical/run_historical_full.go (hunk 4 above)
Comment (derrandz) [reply tests/helpers/block-sync/historical/run_historical_full.go:172]: mistake, thanks for catching this!

File: compositions/cluster-k8s/block-sync/historical/64-square-size/1-3-32-ipld.toml (hunk 2 above)
Comment (derrandz) [reply compositions/cluster-k8s/block-sync/historical/64-square-size/1-3-32-ipld.toml]: yes, removed it. that was my instance, replaced with an informative comment

**Pull request merged**